rules-tests/Php80/Rector/Class_/AnnotationToAttributeRector/Fixture/constant_as_array_key.php.inc

```diff
@@ -9,23 +9,24 @@ use Rector\Tests\Php80\Rector\Class_\AnnotationToAttributeRector\Source\Constant
  * @GenericAnnotation(
  *     itemOperations={
  *         "get" = {
-               ConstantReference::FIRST_NAME = true
+ *              ConstantReference::FIRST_NAME = true,
+ *              'some::string' = 'some-value'
  *         }
  *     }
  * )
  */
 final class DemoClass {
 }
 ?>
------
+    -----
 <?php
 
 namespace Rector\Tests\Php80\Rector\Class_\AnnotationToAttributeRector\Fixture;
 
 use Rector\Tests\Php80\Rector\Class_\AnnotationToAttributeRector\Source\GenericAnnotation;
 use Rector\Tests\Php80\Rector\Class_\AnnotationToAttributeRector\Source\ConstantReference;
 
-#[GenericAnnotation(itemOperations: ['get' => [ConstantReference::FIRST_NAME => true]])]
+#[GenericAnnotation(itemOperations: ['get' => [ConstantReference::FIRST_NAME => true, 'some-string' => 'some-value']])]
```

Comment:
`some::string` to "some-string"? That's seems typo?

Reply:
Yes, sorry! It's fixed now in the PR 🤞